pkg/component/openvpnclient.go

```diff
@@ -78,6 +80,100 @@ func (o OpenVPNClient) getFileMappings() [][]string {
 	}
 }
 
+type RevocationResponse struct {
+	Revoked    bool
+	RevokeTime time.Time
+}
+
+type ListOutput struct {
+	User       string
+	Valid_from string
```

Comment:
Since the project is using camel case everywhere, can you please change this variable name to `ValidFrom`?

Reply:
mmmm in go the standard is snake_case, not camel case btw. Anyway, if this is a super strict requirement, i will change it  :)